lax/lichens/sciencerun1.py

```diff
@@ -52,6 +52,10 @@ class LowEnergyRn220(AllEnergy):
 
     def __init__(self):
         AllEnergy.__init__(self)
+
+        # S2Tails not used in calibration modes
+        self.lichen_list.pop(8)
```

Comment:
Why not just delete the cut from the AllEnergy list above (like in https://github.com/XENON1T/lax/pull/86)?

Reply:
We could. Technically it may also belong in "AllEnergy" since it should work the same at all energy scales.

Reply:
Ah I see sorry, missed that this was in ```LowEnergyRn220```.

Then, should we move this to the bottom, after:
```
self.lichen_list[4] = S2SingleScatterSimple()
```
otherwise the indexing is wrong?